internal/pkg/skuba/deployments/ssh/cri.go

```diff
@@ -35,7 +36,14 @@ func criConfigure(t *Target, data interface{}) error {
 	if err != nil {
 		return errors.Wrap(err, "Could not read local cri directory: "+skuba.CriDir())
 	}
-	defer t.ssh("rm -rf /tmp/cri.d")
+	defer func() {
+		_, _, err := t.ssh("rm -rf /tmp/cri.d")
+		if err != nil {
+			// If the deferred function has any return values, they are discarded when the function completes
+			// https://golang.org/ref/spec#Defer_statements
+			fmt.Println("Could not delete the cri.d config file")
```

Comment:
I would prefer logging to a logger facade here instead of stdout.

```suggestion
			fmt.Println("Could not delete the cri.d config path")
```

Reply:
indeed, do you have a convention on logging yet?

Reply:
@ereslibre :point_up:

Reply:
We are using `klog` for many verbosity-dependant output that prints some level of detail, and `fmt.Print*` family for human friendly messages lacking timestamp and many other details present in log output.

I think we should evaluate merging both together or using two facades for those two different goals. However I think that's out of the scope of this PR.

Reply:
agree with @ereslibre, checking the code the logging is something that we should tackle separately